add theme include colors and text style

AimanYosofiUlt opened the pull request.

Commit:
add theme include colors and text style

ZainabAlnajjar commented:
we won't merge this branch until we finish the sub-issues which would be sub-branches from this one and merge it to this branch first, after all is done then we can merge this branch to dev

Commit:
add empty folder with .gitkeep to keep component package

Commit:
add tudee bottom navigation bar

Commit:
add tudee category task card and priority tag components

Commit:
add spacers

Commit:
add day toggle

EL-MANCY committed:
add the snackBar component handling both dark and light themes using an error flag

Commit:
add day toggle

Commit:
add statue-image and slider components

EL-MANCY committed:
-add the category component handling both dark and light themes
-add an enum class to control the categories
-add Category.kt as an interface to help us with the user added categories

EL-MANCY committed:
edit a comment

diyar214 committed:
finalize the TudeeTabs

Commit:
fix: change a component name from DayToggle to DayItem for better readability
fix: remove duplicated modifier in the DayItem

Commit:
fix a misspelling in the name of the class Statue

Commit:
fix a misspelling in the name of some classes

Commit:
refactor improve readability in the day item class

Commit:
refactor change name for data-chip composable

haideraljawasim committed:
implement text field componet for design system

ZainabAlnajjar committed:
add DarkThemeSwitch component

haideraljawasim committed:
use named argment for color instead of copy and modfiy it

mostafa-n3ma committed:
add , Button Component

EL-MANCY committed:
- create CheckMarkContainer.kt & CategoryCount.kt as a composables

EL-MANCY committed:
change the boolean flag to enum Status.kt class and added description to the SnackBar.kt

mostafa-n3ma committed:
update component parameter

raouf-kamel committed:
Merge pull request #33 from SanaaTeam/component/text-fields

Implement text field componet for design system

ZainabAlnajjar committed:
fix DarkThemeSwitch light bg color

ZainabAlnajjar committed:
fix DarkThemeSwitch animation

Commit:
fix issues in slide class and related classes and resources

Commit:
change using actual date instead of day and day name passed to the DayItem

haideraljawasim committed:
Merge pull request #14 from SanaaTeam/component/nav-bar

Add Bottom Navigation Bar

Commit:
move component package into be inside the design system package

Commit:
move component package into be inside the design system package

Commit:
move component package into be inside the design system package

mostafa-n3ma committed:
update component: enabling the ripple effect

mostafa-n3ma committed:
update spinner animation speed

EL-MANCY committed:
remove redundant package

EL-MANCY committed:
add the modidier as parameter for the SnackBar.kt

EL-MANCY committed:
add shadow fpr the snackbar
edit the padding of the icon

EL-MANCY committed:
removed the Spacers.kt
passed modifier for the the composables and components

raouf-kamel committed:
Merge pull request #18 from SanaaTeam/component/sliders-and-status

Add StatusImage and Slider components

Commit:
refactor DayItem by remove the unnecessary day model

mostafa-n3ma committed:
update , extract button content and apply  previews for all cases

mostafa-n3ma committed:
update , extract button content and apply  previews for all cases

mostafa-n3ma committed:
update , create variables for content text and icons colors

Commit:
refactor category task card and related classes to fix some issues with the code

Commit:
remove spacers from DayItem

EL-MANCY committed:
rename the Status.kt to SnackBarStatus.kt

Commit:
add dimension to the design system and apply it to the current branch

Commit:
Merge branch 'feature/design-system' into component/category-task-pirorty

Commit:
apply dimensions it to the current branch

Commit:
Merge branch 'feature/design-system' into component/day-toggle

Commit:
apply dimensions it to the current branch

EL-MANCY committed:
remove DefaultCategory.kt
make previews private
change image to icon

EL-MANCY committed:
enhance box alignment

AimanYosofiUlt committed:
Merge pull request #15 from SanaaTeam/component/category-task-pirorty

Add Tudee CategoryTaskCard and PriorityTag components

EL-MANCY committed:
rename Category.kt to CategoryUIInfo.kt
use painter instead of a resource Int

AimanYosofiUlt committed:
Merge pull request #17 from SanaaTeam/component/day-toggle

Add DayItem Component 

mostafa-n3ma committed:
add negative buttons

ZainabAlnajjar committed:
improve DarkThemeSwitch animation and code readability

Radwa85 committed:
add BaseBottomSheet composable with flexible height and content

mostafa-n3ma committed:
Merge pull request #35 from SanaaTeam/component/buttons

add , Button Component

Radwa85 committed:
add modifier as parameter to allow customization

ZainabAlnajjar committed:
app AppBar component

ZainabAlnajjar committed:
Merge pull request #20 from SanaaTeam/component/categories

Categories Component

Radwa85 committed:
remove box scope in content parameter

ZainabAlnajjar committed:
Merge remote-tracking branch 'origin/feature/design-system' into component/app-bar-and-switch-button

ZainabAlnajjar committed:
Merge remote-tracking branch 'origin/feature/design-system' into component/app-bar-and-switch-button

Radwa85 committed:
use wrapContentHeight instead of fixed height

ZainabAlnajjar committed:
use theme dimension in AppBar

ZainabAlnajjar committed:
fix error in CategoryTaskCard after merge

AimanYosofiUlt committed:
Merge pull request #46 from SanaaTeam/component/base-bottom-sheet

Component/base bottom sheet

Commit:
add task count by statue card

ZainabAlnajjar committed:
add better resolution app icon

ZainabAlnajjar committed:
fix AppBar onDarkThemeSwitchClick parameter

EL-MANCY committed:
make previews private

ZainabAlnajjar committed:
refactor AppBar to take tailComponent

ZainabAlnajjar committed:
improve readability of DarkModeThemeSwitchButton

ZainabAlnajjar committed:
improve readability of DarkModeThemeSwitchButtonPreview

ZainabAlnajjar committed:
improve memory usage in DarkModeThemeSwitchButton

ZainabAlnajjar committed:
save logoName and appBar captions in String resources

ZainabAlnajjar committed:
Merge pull request #34 from SanaaTeam/component/app-bar-and-switch-button

DarkThemeSwitch component and app bar

diyar214 committed:
remove redundant layouts

diyar214 committed:
update TudeeTabs after review

Commit:
change string resources name

Commit:
removed unused modifiers

refactor TaskCountByStatusCard for better readability

EL-MANCY committed:
Merge remote-tracking branch 'origin/feature/design-system' into component/snack-bar

# Conflicts:
#	app/src/main/res/values/strings.xml

EL-MANCY committed:
Merge pull request #16 from SanaaTeam/component/snack-bar

Snack Bar Component

Commit:
Merge branch 'feature/design-system' into component/status-view-card

# Conflicts:
#	app/src/main/res/values/strings.xml

AimanYosofiUlt committed:
Merge pull request #47 from SanaaTeam/component/status-view-card

Add TaskCountByStatusCard component

diyar214 committed:
Update TudeeTabs.kt

diyar214 committed:
change path of component

diyar214 committed:
Merge branch 'feature/design-system' into component/tabs

diyar214 committed:
update string resources

diyar214 committed:
Merge pull request #42 from SanaaTeam/component/tabs

finalize the TudeeTabs

Commit:
apply design system dimensions

Commit:
refactor the priority tag to be clickable

Commit:
refactor the priority tag to be clickable

Commit:
remove duplicated classes

Commit:
refactor folder structure

Commit:
Merge remote-tracking branch 'origin/dev' into feature/design-system

# Conflicts:
#	gradle/libs.versions.toml

ZainabAlnajjar committed:
remove java import in DateFormater
